### ISSUE TITLE
How to resolve relations between databases

### ISSUE THREAD

**Issue by welyb-alegal** [andrewvo89/notiondb#1]
While fetching database pages, their relations' properties are undefined :

```
const { NotionDB } = require('notiondb');
const { NotionId } = require('notiondb/models');

(async () => {
    const notionDB = new NotionDB(process.env.NOTION_SECRET_TOKEN);
    const dbId = new NotionId(process.env.NOTION_DATABASE_ID);
    const database = await notionDB.databases.get(dbId);
    const pageId = new NotionId(process.env.NOTION_DATABASE_PAGE_ID);
    const page = await database.pages.get(pageId);
    console.log(page.object);
})()
```

```
{
  (...)
  properties: {
    (...),
    "A relation property": undefined,
    (...)
  }
}
```

How can I consolidate data while using relations in my Notion's databases ?

**Comment by andrewvo89** [comment]
Hi @welyb-alegal, thanks for opening the issue.
The current version does not support relation fields, however I can implement this feature within the next 1-2 days.
The output of the relation field would yield an array of page ids, like this:

```json
{
  "properties": {
    "A relation property": {
      "id": "ZhQp",
      "type": "relation",
      "relation": [
        {
          "id": "2fd6fdab-11e1-4ea5-9242-89afa8f6844a"
        },
        {
          "id": "3322572a-e1bf-43c1-80f4-81bd46f9a142"
        },
        {
          "id": "3bc36d4e-1208-440c-8633-3b136c7e52af"
        }
      ]
    }
  }
}
```

**Comment by andrewvo89** [comment]
@welyb-alegal This has now been implemented in version 1.0.13.
I actually decided to return the references in an array of strings like so:

```json
{
  "properties": {
    "A relation property": {
      "id": "ZhQp",
      "type": "relation",
      "relation": [
        "2fd6fdab-11e1-4ea5-9242-89afa8f6844a",
        "3322572a-e1bf-43c1-80f4-81bd46f9a142",
        "3bc36d4e-1208-440c-8633-3b136c7e52af"
      ]
    }
  }
}
```
Once you have the relation IDs, you can do a lookup (Page get()) on a separate database instance (the relation database), to get those pages.

The same input would be for updating references to another notion page:
```javascript
 const newPage = await page.update({
    relation: [
      "2fd6fdab-11e1-4ea5-9242-89afa8f6844a",
      "3322572a-e1bf-43c1-80f4-81bd46f9a142",
      "3bc36d4e-1208-440c-8633-3b136c7e52af",
    ],
  });
  console.log(newPage.object);
```

Just make sure that your referenced pages are given access to your API Integration key, otherwise they will not appear in the array when retrieving the relation property.

**Comment by welyb-alegal** [comment]
Thanks @andrewvo89 I figured it out myself almost the same way you did.

Nice job
